Week3/homework/js-exercises/3-fortune.teller.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+function tellFortune(numChildren, partnerNames, locations, jobs){
+  numChildren = [1, 2, 3, 4, 5];
+  partnerNames = [ "A", "B", "C", "D", "E"];
+  locations = ["Istanbul", "Amsterdam", "Paris","Diyarbakir", "Tokyo"];
+  jobs = ["waiter", "doctor", "lawyer", "painter", "singer"];
```

Comment:
Ah. I think the homework requires you to move these declarations outside the function and then pass them as arguments to it when invoking at line 13. Could you please correct it? Also, make sure you use `const` keyword instead of `let` while declaring them.